main.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'app'
+
+class Main
+  def initialize
+    @console = ConsoleApp.new
+  end
+
+  def all_books
+    @console.all_books
+  end
+
+  def all_people
+    @console.all_people
+  end
+
+  def add_person
+    @console.add_person
+  end
+
+  def add_book
+    @console.add_book
+  end
+
+  def add_rental
+    @console.add_rental
+  end
+
+  def list_rentals_for_person
+    @console.list_rentals_for_person
+  end
+
+  def main
+    puts '\nLibrary Menu:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
```

Comment:
- Hey, when this option is being selected, it should as the user if they want to create a Student or Teacher according to the demo, however, it doesn't show that behavior, it returns `Enter person's name: ` after entering the name, it returns 
`Created person: unknown`. Please, kindly fix this issue. check the demo video and project requirements to fix this issue.

---

NOTE: Since the app currently is not getting any person created, one can't further access the remaining options as the remaining options need a person to be created before one can 
`5. Create a rental 6. List rentals for a person`. Please, ensure all these options work correctly before requesting for another review, as they will be tested to ensure they meet the project requirement. :muscle:

Reply:
- [ ] As requested by the previous reviewer, your app is NOT behaving like the one in the video

Your App
![image](https://github.com/boniface193/oop_ruby/assets/81932478/22fae65c-658a-4a14-a67e-bc12715edec3)

Given App
![image](https://github.com/boniface193/oop_ruby/assets/81932478/10e5961f-fdeb-4b25-9dad-e7f8dface5f8)

#### Note that the app should ask the user to select with 1 for creating a student and 2 for a teacher


- [ ] Again as requested you will have to ensure your app operates as the one in the given [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). Since no person is being created its difficult to check other functionalities. You will have to make sure that a User is able to create both a Student and Teacher. You will also have to make sure user is able to create a rental as is in the provided video
![image](https://github.com/boniface193/oop_ruby/assets/81932478/650483fc-f581-4821-a018-8ac4a7780a73)